plugins/mentions/mentions.py

```diff
@@ -12,11 +12,21 @@
 class Config(BaseProxyConfig):
     def do_update(self, helper: ConfigUpdateHelper) -> None:
         helper.copy("groups_config_src")
+        helper.copy("sync_config_interval")
 
 
-class Mentions(Plugin):
-    SYNC_CONFIG_FREQUENCY = 300  # secs
+# returns default bool if all elements in boolList are not same
+def get_collective_behavior(boolList: List[bool], default: bool) -> bool:
+    if len(boolList) == 1:
+        return boolList[0]
+
+    if all(value == boolList[0] for value in boolList):
```

Comment:
Is it intended to always compare to the first entry in `boolList`? I'm not sure I understand what's going on here.

Reply:
Simply checking if all values are same. So checking all values against the first one.